lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -1080,7 +1123,7 @@ class _MouseAdapter extends _BaseAdapter with _WheelEventListenerMixin {
       _callback(pointerData);
     });
 
-    _addMouseEventListener(domWindow, 'mousemove', (DomMouseEvent event) {
+    _addMouseEventListener(glassPaneElement, 'mousemove', (DomMouseEvent event) {
```

Comment:
It's explained here why this needs to be attached to `window`:
https://github.com/flutter/engine/blob/13ae6eb75ab97947b2bcd507bf45ee33a1d2033d/lib/web_ui/lib/src/engine/pointer_binding.dart#L283-L286

Changing it to listen on `glassPaneElement` will break some use cases.

Reply:
AFAIK the MouseAdapter is only being used by Safari 12 and older. Also I'm struggling to understand the comment here.

Is this comment saying that the browser is FIRING `mousemove` events when the cursor is OUTSIDE of `window`? I don't believe this comment one bit :P

Anyway, can revert no problem :)

Reply:
I think when you listen on window, you receive some events happening outside the window (e.g. when you start dragging inside the window, then move the mouse outside).

You lose that when you listen on an element.

But my memory is vague here, so I could be wrong.

Reply:
Since this only affects the mouse adapter (that's going away soon), I'm not worried about the effects of this change anymore. So feel free to leave it attached to glassPaneElement.

Reply:
You're right, this is shocking!
https://jsfiddle.net/ditman/7towxaqp

Reply:
I've reverted the code so the event is attached to the window so we keep seeing those events outside of the window. This might be weird if we want to drag-and-drop across two flutter views at some point :/